block-layer/content-addressable-archives.md

```diff
@@ -112,32 +171,36 @@ The first two bytes of a multihash are varints, where the second varint is an un
 
 The remainder of a Section, after length-prefix and CID, comprises the raw byte data of the IPLD block. The encoded block may be any IPLD block format as specified by the codec in the CID. Typical codecs will be [DAG-PB](codecs/dag-pb.md), [DAG-CBOR](codecs/dag-cbor.md) or [RAW](https://github.com/ipld/specs/issues/223).
 
-## Additional Considerations
+Duplicate blocks _must_ not be present in a deterministic CAR, and _should_ not be present in a non-deterministic CAR.
 
-### Determinism
+## Deterministic CAR
 
-Deterministic CAR creation is not covered by this specification. However, deterministic generation of a CAR from a given graph is possible and is relied upon by certain uses of the format, most notably, [Filecoin](https://filecoin-project.github.io/specs).
+Where a `deterministic` value of `true` is present in a CAR, the header is able to describe the entirety of the graph represented by the trailing blocks in the _Data_ section. Such a CAR may be reproduced, byte-for-byte, from a block source using the same roots and selectors.
 
-Additional rules for the generation of the CAR format may be applied in order to ensure that the same CAR is always generated from the same data. The burden of this determinism is primarily placed on [selectors](../selectors/selectors.md) whereby a given selector applied to a given graph will always yield blocks in the same order regardless of implementation.
+The burden of this determinism is primarily placed on [selectors](../selectors/selectors.md) whereby a given selector applied to a given graph will always yield blocks in the same order regardless of implementation and the boundaries of that graph will be exhaustively described by the selector. Partial graphs may require very complex selectors to describe while retaining determinism.
 
-Care regarding the ordering of the `roots` array in the Header, as well as consideration for CID version _(see [below](#cid-version))_ and avoidance of duplicate blocks _(see [below](#duplicate-blocks))_ may also be required for strict determinism.
+Where multiple roots and their selectors are present in the _Header_ of a deterministic CAR, the blocks constituting their graphs are presented in the same order as the roots are present in the _Header_.
 
-All such considerations are deferred to the user of the CAR format and should be documented there as this specification does not inherently support determinism.
+Duplicate blocks are strictly not valid in a deterministic CAR. The first instance of a block discovered by any root + selector combination will be present, all repeated occurances of the same block encountered during further graph traversal are omitted.
+
+### Additional Considerations
```

Comment:
I think this may have meant to be an h2?

(I like using the two-line syntax variant for these because it's much more visually distinctive and easier to catch myself in typos like this, fwiw :upside_down_face: )